MANIFEST.in

```diff
@@ -1,12 +1,7 @@
-include dash_core_components/dash_core_components.min.js
-include dash_core_components/dash_core_components.min.js.map
-include dash_core_components/dash_core_components-shared.js
-include dash_core_components/dash_core_components-shared.js.map
-include dash_core_components/async-*.js
-include dash_core_components/async-*.js.map
+include dash_core_components/*.js
+include dash_core_components/*.js.map
```

Comment:
This is consistent with what is being done in other repos and how these deps are published in the npm package.